pennylane_lightning/core/src/measurements/tests/Test_MeasurementsBase.cpp

```diff
@@ -217,10 +217,12 @@ template <typename TypeList> void testProbabilitiesObs() {
 
         // Defining the Statevector that will be measured.
         auto statevector_data = createNonTrivialState<StateVectorT>();
+        auto sv_data = createNonTrivialState<StateVectorT>();
```

Comment:
If the goal is to create the same state, set to `statevector_data` so if the last changes this one will follow along.
```suggestion
        auto sv_data = statevector_data;
```

Reply:
Thanks @vincentmr ! The suggested change will work for all backends except for `LQRaw`. With `LQRaw`, both measurement objects will operate on the same data, `statevector_data` , if we set `sv_data = statevector_data`. In this case, I will keep current implementation.

Reply:
It's a nitpick so I'm fine either way. But `createNonTrivialState` is returning a `vector` so `sv_data = statevector_data` should create a copy which is compatible with `LQRaw`, unlike passing `statevector_data.data()` into both SV constructors.